ui-tests/cypress/e2e/pages/templates.cy.js

```diff
@@ -506,7 +506,7 @@ describe('Page Templates', () => {
         });
     });
 
-    it([Tag.SANITY, 'ENG-3525', 'ENG-3664'], 'When confirming deletion, the list should be updated and a successful toast notification displayed', () => {
+    it([Tag.SANITY, 'ENG-3525', 'ENG-3664', 'ENG-3984'], 'When confirming deletion, the list should be updated and a successful toast notification displayed', () => {
```

Comment:
ENG-3664 is the bug for v7; please remove the tag leaving only the tag for the v6 bug (i.e. ENG-3984)